MaeBlock.cpp

```diff
@@ -242,6 +243,7 @@ EXPORT_MAEPARSER void IndexedBlock::setProperty<string>(
 size_t IndexedBlock::size() const
 {
     size_t count = 0;
+    // Counts of these maps are 1:1 with properties in the map
```

Comment:
I don't know that this clarifies it. Why does it check length of all of them? Why isn't one good enough?

Reply:
Wrote a much longer comment